mesh_tools/create_SCRIP_files/create_SCRIP_file_from_CISM_mesh.py

```diff
@@ -122,23 +166,33 @@
 grid_corner_lat[:] = grid_corner_lat_local[:]
 
 grid_imask[:] = 1  # For now, assume we don't want to mask anything out - but eventually may want to exclude certain cells from the input mesh during interpolation
-grid_dims[:] = (nx * ny)
 
+##HH
+if int(options.gridRank) == 1:
+    print('grid dims is nx*xy')
+    grid_dims[:] = (nx * ny) # for RANK 1 data
+elif int(options.gridRank) == 2:
+    print('grid dims is nx,xy')
+    grid_dims[:] = [nx , ny]
```

Comment:
It's always best to have `if`/`elseif`/`else`.  This way, there is a clear error message when a wrong parameter value is given:
```suggestion
if int(options.gridRank) == 1:
    print('grid dims is nx*xy')
    grid_dims[:] = (nx * ny) # for RANK 1 data
elif int(options.gridRank) == 2:
    print('grid dims is nx,xy')
    grid_dims[:] = [nx , ny]
else:
    raise ValueError(f'gridRank must be 1 or 2 but {options.gridRank} was given.') 
```

Reply:
Oh, I see, you do this above...